.github/workflows/test-m1.yml

```diff
@@ -7,7 +7,13 @@ on:
     branches:
       - nightly
       - main
+    tags:
+        # NOTE: Binary build pipelines should only get triggered on release candidate builds
+        # Release candidate tags look like: v1.11.0-rc1
+        - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
```

Comment:
Why not just add `- release/*` branches to the list? I.e. as tags must point to commits on `release/` branch, tagged tests would be no different then regular CI for pushes to the branch?

Reply:
Done

Reply:
No, it's not (tags are still there)

Reply:
Sorry I misunderstood your first comment, added the release branch but thought we still need tests on rc